front-end-poc/README.md

```diff
@@ -5,39 +5,49 @@ This project is a PoC for the Audit section of a XAI service
 ### Install dependencies
 
 To install dependencies you need to have yarn installed globally and run in the terminal:
+
 ```
 yarn install
 ```
+
+### Install GWT editors
+
+To install the GWT editors to view DMN models you need to run the following in the terminal before running the project:
+
+```
+yarn run prepare
```

Comment:
Is that true? the prepare script should be run automatically after `yarn install`. At least it worked last time I tested.

Did you find any issue with that?

Reply:
Thank-you for highlighting @kelvah.

TBH IDK `prepare` was a pre-defined life-cycle script. `README.md` updated in new commit.